hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/QueriesConfig.java

```diff
@@ -16,8 +16,10 @@
 
 package com.hedera.node.config.data;
 
+import com.hedera.node.config.NodeProperty;
 import com.swirlds.config.api.ConfigData;
 import com.swirlds.config.api.ConfigProperty;
 
 @ConfigData("queries")
-public record QueriesConfig(@ConfigProperty(value = "blob.lookupRetries", defaultValue = "3") int blobLookupRetries) {}
+public record QueriesConfig(
+        @ConfigProperty(value = "blob.lookupRetries", defaultValue = "3") @NodeProperty int blobLookupRetries) {}
```

Comment:
Not sure what this is, but it seems like it might be more a network property.

Reply:
I wonder if it is still used? Once upon a time files (blobs) were stored in postgres and I assume this property existed to retry N times, just in case something funky happened. If we do still use it, a node property is probably OK, since it won't be deterministic across nodes (i.e. if I have a setting of 3 and you have a setting of 5, it probably has little impact on whether we ISS or not).

Reply:
Deleted QueriesConfig since there is only one property here and not used.